rakam-spi/src/main/java/org/rakam/analysis/InMemoryMetastore.java

```diff
@@ -78,4 +74,9 @@ public void deleteProject(String project) {
         apiKeyService.revokeAllKeys(project);
         super.onDeleteProject(project);
     }
+
+    @Override
+    public List<String> getAttributes(String project, String collection, SchemaField field, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<String> query) {
+        return null;
```

Comment:
Instead of returning `null`, we should throw `new UnsupportedOperationException()` in Clickhouse, Dynamodb and also here.

Reply:
Done